### ISSUE TITLE
Improve check of array identation

### ISSUE THREAD

**Issue by xalopp** [mayflower/mo4-coding-standard#66]
PHPCS 3.2.0 adds the new Sniff `Generic.Arrays.ArrayIndent`  which checks if the array elements are properly indented.

valid:

```
$a = [
    'a',
];
```

invalid but currently not reported:
```
$a = [
'a',
];
```
